common/lib/xmodule/xmodule/modulestore/xml_exporter.py

```diff
@@ -89,7 +89,7 @@ def _export_drafts(modulestore, course_key, export_fs, xml_centric_course_key):
                 parent = modulestore.get_item(draft_node.parent_location)
 
                 # Don't try to export orphaned items
-                if draft_node.module.location not in parent.children:
+                if not hasattr(parent, 'children') or draft_node.module.location not in parent.children:
```

Comment:
So this change happened because I started seeing some flakey test errors when running the `test_mongo.py` tests (specifically the `test_export_course_image` and `test_export_course_image_nondefault` tests). The error would show up as 
```
if draft_node.module.location not in parent.children:
AttributeError: 'HiddenDescriptorWithMixins' object has no attribute 'children'
```
But I have no idea how or why it would present itself in this way. It feels like maybe something breaks during the import step (see above in this same file during `setUpClass`), but I still don't understand how it results in this output. 

This change felt like a safe protection regardless as it is just ensuring the attribute exists prior to accessing it, but happy to hear thoughts here.